src/main/java/com/nha/abdm/wrapper/hip/hrp/database/mongo/services/RequestLogService.java

```diff
@@ -380,4 +384,44 @@ public void updateConsentResponse(String requestId, ConsentStatus consentDetails
     update.set(FieldIdentifiers.STATUS, RequestStatus.CONSENT_ON_STATUS_RESPONSE_RECEIVED);
     mongoTemplate.updateFirst(query, update, RequestLog.class);
   }
+
+  public void dataTransferNotify(
```

Comment:
I see in @atulai-sg's implementation that the `Consent`s are saved in `Patient` collection. Can you please consolidate this?